src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java

```diff
@@ -112,10 +118,13 @@ final boolean wasExecuted() {
         return null != tdsReader;
     }
 
+    /** Return parameter for stored procedure calls */
+    Parameter returnParam;
```

Comment:
SonarQube has a critical error - Fields in a Serializable class must themselves be either Serializable or transient. Is this possible?
```suggestion
    transient Parameter returnParam;
```